requirements/base.txt

```diff
@@ -2,7 +2,7 @@ chevron~=0.12
 click~=7.1
 Flask~=1.1.2
 #Need to add Schemas latest SDK.
-boto3~=1.18
+boto3>=1.18.32,==1.*
```

Comment:
Any reason we choose `==1.*` over `<2`? I think they are equivalent but was hard to reason about in my head.

Reply:
No difference. Chose this format as it follows the expanded ~= syntax.
PEP 440: https://www.python.org/dev/peps/pep-0440/#compatible-release